includes/admin/transifex-live-integration-admin-template.php

```diff
@@ -46,6 +46,11 @@
 				<input name="transifex_live_settings[translate_urls]" type="checkbox" id="transifex_live_settings_translate_urls" value="1" <?php echo $checked_translate_urls ?>><?php _e( 'Make images and links translatable' ); ?>&nbsp;<a target="_blank" href="<?php echo $settings['urls']['translate_urls']; ?>"><b><?php _e( 'Learn more' ) ?></b></a>.
 			</td>
 		</tr>
+		<tr>
+			<td style="padding-right:0px">
+				<input name="transifex_live_settings[canonical_urls]" type="checkbox" id="transifex_live_settings_canonical_urls" value="1" <?php echo $checked_canonical_urls ?>><?php _e( 'Disable rendering canonical URLs' ); ?>&nbsp;<a target="_blank" href="<?php echo $settings['urls']['canonical_urls']; ?>"><b><?php _e( 'Learn more' ) ?></b></a>.
+			</td>
+		</tr>
```

Comment:
I think there something wrong with the spacing here (see ![image](https://github.com/transifex/transifex-live-wordpress/assets/81964427/e1488282-0ca6-4964-9233-56b80a30c98c)). Was it intentional?